roles/openshift_gce_cluster/templates/remove.j2.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
```

Comment:
Consider an ansible module for these shell script type functions/scripts

Reply:
In general, I think an ansible module would actually be worse / less clear.  The reason this is structured as it is is because gcloud is the practical public API for these calls.  Switching to python modules now would be more work and complexity with less guarantee of success, and less flexibility.  I do think the long term arc for this would be to use deployment manager and other pieces, but I think a naive port to an ansible module would be harder to maintain, more code, more prone to bugs, and less clear.   Over time, I'd expect GCE modules to improve, but today they cover maybe a tenth of the things that this will.

I'll add some write up into these about why they are the way they are, then highlight how this could be moved to deployment manager.  This is a proven correct, fast, and resilient deployment setup.  It should be incrementally moved forward in small steps.